tools/cpackget/docs/cpackget.md

```diff
@@ -19,101 +27,211 @@ Usage:
 
 Available Commands:
   help        Help about any command
-  index       Updates public index
-  init        Initializes a pack root folder
-  pack        Adds/Removes Open-CMSIS-Pack packages
-  pdsc        Adds/Removes Open-CMSIS-Pack packages in the local file system via PDSC files.
+  index       Update public index
+  init        Initialize a pack root folder
+  add         Add Open-CMSIS-Pack packages
+  rm          Remove Open-CMSIS-Pack packages
+  list        List installed packs
 
 Flags:
   -h, --help               help for cpackget
-  -R, --pack-root string   Specifies pack root folder. Defaults to CMSIS_PACK_ROOT environment variable
+  -R, --pack-root path     Specifies path of the central CMSIS-Pack root directory, default: CMSIS_PACK_ROOT environment variable
   -q, --quiet              Run cpackget silently, printing only error messages
   -v, --verbose            Sets verboseness level: None (Errors + Info + Warnings), -v (all + Debugging). Specify "-q" for no messages
   -V, --version            Prints the version number of cpackget and exit
 
 Use "cpackget [command] --help" for more information about a command.
 ```
 
-For example, if one wanted help removing a pack, running `cpackget pack rm --help` would print out useful information on the subject.
+Usage help for a specific command available, with for example: `cpackget rm --help`.
+
+## Specify central CMSIS-PACK directory
+
+`cpackget` is compatible with other CMSIS-Pack management tools, such as the Pack Installer available in MDK or Eclipse variants. There are two ways to specify the central CMSIS-PACK root directory:
+
+1. With the `CMSIS_PACK_ROOT` environment variable. Refer to [Installation - Environment Variables](Installation.md#environment-variables).
+
+2. with the option `--pack-root <path>`, for example:
 
-### Specify the working pack root folder
+   ```yml
+   $ cpackget add Vendor.PackName -path-root /MyLocal/Packs
```

Comment:
--pack-root instead of -path-root
./MyLocal/Packs (I assume you were specifying a relative path)